xml/System.Net.Http/HttpClient.xml

```diff
@@ -2662,10 +2662,26 @@ The <paramref name="requestUri" /> is not an absolute URI.
         <Parameter Name="request" Type="System.Net.Http.HttpRequestMessage" Index="0" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="request">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="request">The HTTP request message to send.</param>
+        <summary>Send an HTTP request as a synchronous operation.</summary>
+        <returns>HTTP response message.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ This operation does block until entire response including content is read or `cancellationToken` parameter cancels the operation.
+
+> [!NOTE]
+> In case of timeout, different exceptions are thrown on different .NET implementations.
+> - <xref:System.Net.Http.HttpRequestException> is thrown on all applicable .NET Framework versions.
+> - <xref:System.Threading.Tasks.TaskCanceledException> without any inner exception is thrown on all applicable .NET Core versions.
+> - <xref:System.Threading.Tasks.TaskCanceledException> that nests a <xref:System.TimeoutException> is thrown on .NET 5.0 and later versions.
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.ArgumentNullException">The <paramref name="request" /> is <see langword="null" />.</exception>
```

Comment:
Also `NotSupportedException`:
- for HTTP version >= 2.0 or when requesting version upgrade via `HttpVersionPolicy.RequestVersionOrHigher`
- if using custom `HttpContent` not overriding synchronous `SerializeToStream`
- if using custom `HttpMessageHandler` not overriding synchronous `Send`